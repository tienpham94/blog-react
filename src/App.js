import React from 'react'

import 'normalize.css'
import styles from './App.css'

const App = () => (
  <h1 className={styles.heading}>Hello World!</h1>
)

export default App
