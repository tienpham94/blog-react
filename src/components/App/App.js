import React from 'react'
import { BrowserRouter } from 'react-router-dom'

import 'normalize.css'
import styles from './App.css'
import Header from '../Header/Header'
import Routes from '../Routes/Routes'

const App = () => (
  <BrowserRouter>
    <div>
      <Header />
      <div className={styles.container}>
        <Routes />
      </div>
    </div>
  </BrowserRouter>
)

export default App
