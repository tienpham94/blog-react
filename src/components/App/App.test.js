import React from 'react'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import App from './App'

test('sample', () => {
  const component = shallow(<App />)
  const tree = toJson(component)

  expect(tree).toMatchSnapshot()
})
