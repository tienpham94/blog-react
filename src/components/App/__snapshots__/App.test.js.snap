exports[`test sample 1`] = `
<BrowserRouter>
  <div>
    <Header />
    <div
      className="container">
      <Routes />
    </div>
  </div>
</BrowserRouter>
`;
